packages/shared/src/domain/rate-limiting/index.ts

```diff
@@ -25,22 +26,22 @@ export const rateLimitCountSchema = z.object({
   resetTime: z.number().optional(), // https://express-rate-limit.mintlify.app/guides/creating-a-store
 });
 
-export const routeMapForError: Record<RateLimitOperation, string> = {
+export const errorMessageByOperation: Record<RateLimitOperation, string> = {
   patientQuery: "Too many patient creates or updates, please try again later.",
-  documentQuery: "Too many patient documeny query starts, please try again later.",
+  documentQuery: "Too many patient document query starts, please try again later.",
   consolidatedDataQuery: "Too many patient consolidated data query starts, please try again later.",
 };
 
-export const defaultOperationLimits: {
-  [k in RateLimitOperation]: { [k in RateLimitWindow]: number };
+export function getDefaultLimit(operation: RateLimitOperation): number {
+  const limit = defaultOperationLimits[operation];
+  if (!limit) throw new MetriportError("Limit not found", undefined, { operation });
```

Comment:
nit: maybe add "rate" to the message to help separate it from any other place we might use just "limit"?

Reply:
Done.